src/Modal.svelte

```diff
@@ -74,8 +77,14 @@
     state = { ...defaultState, ...options };
     onOpen = callback.onOpen || toVoid;
     onClose = callback.onClose || toVoid;
-    onOpened = callback.onOpened || toVoid;
-    onClosed = callback.onClosed || toVoid;
+    onOpened = () => {
+      callback.onOpened || toVoid;
```

Comment:
This is doing nothing now.

Should be changed to:
```suggestion
      if (callback.onOpened) callback.onOpened(event);
```